<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Wowchemy 5.3.0 for Hugo"><meta name=description content><link rel=alternate hreflang=en-us href=https://bspl.korea.ac.kr/interesting_works/><link rel=preconnect href=https://fonts.gstatic.com crossorigin><meta name=theme-color content="#1565c0"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin=anonymous media=print onload='this.media="all"'><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css integrity="sha256-FMvZuGapsJLjouA6k7Eo2lusoAX9i0ShlWFG6qt7SLc=" crossorigin=anonymous media=print onload='this.media="all"'><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/github.min.css crossorigin=anonymous title=hl-light media=print onload='this.media="all"'><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/dracula.min.css crossorigin=anonymous title=hl-dark media=print onload='this.media="all"' disabled><link rel=preload as=style href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap"><link rel=stylesheet href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap" media=print onload='this.media="all"'><link rel=stylesheet href=/css/wowchemy.d4217de0a97b753d87af942d59c81115.css><link rel=alternate href=/interesting_works/index.xml type=application/rss+xml title="Brain Signal Processing Lab"><link rel=manifest href=/manifest.webmanifest><link rel=icon type=image/png href=/media/icon_hu746e86b219933ead05b970d8e60fbf48_161589_32x32_fill_lanczos_center_3.png><link rel=apple-touch-icon type=image/png href=/media/icon_hu746e86b219933ead05b970d8e60fbf48_161589_180x180_fill_lanczos_center_3.png><link rel=canonical href=https://bspl.korea.ac.kr/interesting_works/><meta property="twitter:card" content="summary"><meta property="og:site_name" content="Brain Signal Processing Lab"><meta property="og:url" content="https://bspl.korea.ac.kr/interesting_works/"><meta property="og:title" content="Interesting Works | Brain Signal Processing Lab"><meta property="og:description" content><meta property="og:image" content="https://bspl.korea.ac.kr/media/icon_hu746e86b219933ead05b970d8e60fbf48_161589_512x512_fill_lanczos_center_3.png"><meta property="twitter:image" content="https://bspl.korea.ac.kr/media/icon_hu746e86b219933ead05b970d8e60fbf48_161589_512x512_fill_lanczos_center_3.png"><meta property="og:locale" content="en-us"><meta property="og:updated_time" content="2023-03-08T00:00:00+00:00"><title>Interesting Works | Brain Signal Processing Lab</title></head><body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents class=page-wrapper data-wc-page-id=6968ecbcb5e980dded5666bc95f80948><script src=/js/wowchemy-init.min.1d309c6b3f55725f8869af2651084961.js></script><aside class=search-modal id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=# aria-label=Close><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control aria-label=Search...></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><div class=page-header><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container-xl><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=/>Brain Signal Processing Lab</a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar-content aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=/>Brain Signal Processing Lab</a></div><div class="navbar-collapse main-menu-item collapse justify-content-start" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=/aims><span>BSPL is..</span></a></li><li class=nav-item><a class=nav-link href=/members><span>Members</span></a></li><li class="nav-item dropdown"><a href=# class="nav-link dropdown-toggle" data-toggle=dropdown aria-haspopup=true><span>Publications</span><span class=caret></span></a><div class=dropdown-menu><a class=dropdown-item href=/publications/articles><span>Articles</span></a>
<a class=dropdown-item href=/publications/domestic_conferences><span>Domestic Conferences</span></a>
<a class=dropdown-item href=/publications/international_conferences><span>International Conferences</span></a></div></li><li class="nav-item dropdown"><a href=# class="nav-link dropdown-toggle" data-toggle=dropdown aria-haspopup=true><span>Softwares</span><span class=caret></span></a><div class=dropdown-menu><a class=dropdown-item href=/softwares/dnn><span>DNN</span></a>
<a class=dropdown-item href=/softwares/idr><span>iDR</span></a>
<a class=dropdown-item href=/softwares/rspca><span>rsPCA</span></a>
<a class=dropdown-item href=/softwares/env><span>ENV</span></a></div></li><li class=nav-item><a class="nav-link active" href=/interesting_works><span>Interesting Works</span></a></li><li class=nav-item><a class=nav-link href=/news><span>Lab News</span></a></li><li class=nav-item><a class=nav-link href=/contact><span>Contact Us</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class=nav-item><a class="nav-link js-search" href=# aria-label=Search><i class="fas fa-search" aria-hidden=true></i></a></li><li class="nav-item dropdown theme-dropdown"><a href=# class=nav-link data-toggle=dropdown aria-haspopup=true aria-label="Display preferences"><i class="fas fa-moon" aria-hidden=true></i></a><div class=dropdown-menu><a href=# class="dropdown-item js-set-theme-light"><span>Light</span></a>
<a href=# class="dropdown-item js-set-theme-dark"><span>Dark</span></a>
<a href=# class="dropdown-item js-set-theme-auto"><span>Automatic</span></a></div></li></ul></div></nav></div><div class=page-body><div class="universal-wrapper pt-3"><h1>Interesting Works</h1></div><div class=universal-wrapper><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/interesting_works/2021_11_17_bai_et_al_ieee/>[Article] Biomarker identification through integrating fmri and epigenetics</a></div><div class=mt-2><p>Summary: They combined linear regression with CCA in a coupled manner to extract discriminative features for schizophrenia that are co-expressed in the fMRI and DNA methylation data (epigenetic data).</p><p><a href=https://ieeexplore.ieee.org/abstract/document/8786218 target=_blank rel=noopener>Bai, Yuntong, et al. &ldquo;Biomarker identification through integrating fmri and epigenetics.&rdquo; IEEE transactions on Biomedical Engineering 67.4 (2019): 1186-1196.</a><br></p></div><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>Nov 17, 2021</span></div></div></div><div class=ml-3></div></div><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/interesting_works/2021_11_10_gong_et_al_naturecommun/>[Article] Brain structure is linked to the association between family environment and behavioral problems in children in the ABCD study</a></div><div class=mt-2><p>Summary: In this article, they tried to find a relationship between family conflict, parental monitoring and children&rsquo;s behavior problems and cognitive scores. They utilized sMRI, CBCL and KSADS to get 20 behavioral problems scores and 10 cognitive scores. They visualized correlation between these criteria and showed structural differences according to these scores. Furthermore, they performed longitudinal association analysis with baseline & 1 year later data. Utilizing CBCL scores into account in my study later seems reasonable.</p><p><a href=https://www.nature.com/articles/s41467-021-23994-0 target=_blank rel=noopener>Gong, et al. Brain structure is linked to the association between family environment and behavioral problems in children in the ABCD study. Nat Commun 12, 3769 (2021)</a><br></p></div><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>Nov 10, 2021</span></div></div></div><div class=ml-3></div></div><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/interesting_works/2021_11_03_dinsdale_et_al_neuroimage_2021/>[Article] Learning patterns of the ageing brain in MRI using deep convolutional networks</a></div><div class=mt-2><p>Summary: Summary: In this paper, the authors predicted the brain age of T1-weighted MRI images in UK Biobank dataset using 3D CNN architecture. They examined the relationship between predicted ages and UK Biobank variables which are categorized into lifestyle factors (exercise, alcohol, tobacco, &mldr;), physiological/medical measurements (bone, cardiac, eye, &mldr;), and medical history. They adopted attention gates as attention modules to observe activation of the CNN model, and they also visualized the model using saliency map for comparison.</p><p><a href=https://www.sciencedirect.com/science/article/pii/S1053811920308867?via%3Dihub target=_blank rel=noopener>Dinsdale, Nicola K., et al. &ldquo;Learning patterns of the ageing brain in MRI using deep convolutional networks.&rdquo; Neuroimage 224 (2021): 117401.</a><br></p></div><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>Nov 3, 2021</span></div></div></div><div class=ml-3></div></div><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/interesting_works/2021_10_27_lim_et_al_neuroiamge_2021/>[Article] Dynamic functional connectivity markers of objective trait mindfulness.</a></div><div class=mt-2><p>Summary: The goal of study is to identify the differences in dynamic functional connectivity according to the trait mindfulness. They found the high mindfulness trait group showed more strong within-network connectivity in the DMN and the salience network than the low mindfulness trait group.</p><p><a href=https://www.sciencedirect.com/science/article/pii/S1053811918303665 target=_blank rel=noopener>Lim, J., Teng, J., Patanaik, A., Tandi, J., & Massar, S. A. (2018). Dynamic functional connectivity markers of objective trait mindfulness. NeuroImage, 176, 193-202.</a><br></p></div><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>Oct 27, 2021</span></div></div></div><div class=ml-3></div></div><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/interesting_works/2021_10_13_dinsdale_et_al_journalofneurosciencemethods_2021/>[Article] Integrated 3D motion analysis with functional magnetic resonance neuroimaging to identify neural correlates of lower extremity movement.</a></div><div class=mt-2><p>Summary: Collected knee biomechanics.using an MRI-compatible motion capture system. The study shows promise for the MRI-compatible system to capture lower-extremity biomechanical data collected concurrently during fMRI, and the present data identified potentially unique neural drivers of aberrant biomechanics.</p><p><a href=https://www.sciencedirect.com/science/article/pii/S0165027021000431 target=_blank rel=noopener>Anand, M., Diekfuss, J. A., Slutsky-Ganesh, A. B., Grooms, D. R., Bonnette, S., Foss, K. D. B., &mldr; & Myer, G. D. (2021). Integrated 3D motion analysis with functional magnetic resonance neuroimaging to identify neural correlates of lower extremity movement. Journal of Neuroscience Methods, 355, 109108.</a><br></p></div><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>Oct 13, 2021</span></div></div></div><div class=ml-3></div></div><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/interesting_works/2021_10_06_dinsdale_et_al_neuroimage_2021/>[Article] Deep learning-based unlearning of dataset bias for MRI harmonisation and confound removal.</a></div><div class=mt-2><p>Summary: The authors proposed a deep learning based training scheme, inspired by domain adaptation techniques. It is similar to my current study, but this study proposed a consecutive training process since they assumed that they may not know main task label (i.e., prediction target such as age, gender, p-factor) even they can know all available domain label (e.g., site, scanner). It would be good to present this paper because this paper has a similar, but different point of view for dealing with the scanner-effect.</p><p><a href=https://www.sciencedirect.com/science/article/pii/S1053811920311745 target=_blank rel=noopener>Dinsdale, N. K., Jenkinson, M., & Namburete, A. I. (2021). Deep learning-based unlearning of dataset bias for MRI harmonisation and confound removal. NeuroImage, 228, 117689.</a><br></p></div><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>Oct 6, 2021</span></div></div></div><div class=ml-3></div></div><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/interesting_works/2021_09_29_bao_etal_nature_2020/>[Article] A map of object space in primate inferotemporal cortex.</a></div><div class=mt-2><p>Summary: The authors investigated the (category-selective) organization of macaque IT cortex using fMRI, microstimulation, electrophysiology. A total of 51 objects each presented at 24 views belonging to 6 different categories. The fc6 output of AlexNet for object classification was PCA decomposed and used as an object space. Pooling responses across face, body, NML, and stubby networks enabled reasonable reconstruction of general objects demonstrating the importance of these four networks in object representation.</p><p><a href=https://www.nature.com/articles/s41586-020-2350-5 target=_blank rel=noopener>Bao, P., She, L., McGill, M., & Tsao, D. Y. (2020). A map of object space in primate inferotemporal cortex. Nature, 583(7814), 103-108.</a><br></p></div><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>Sep 29, 2021</span></div></div></div><div class=ml-3></div></div><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/interesting_works/2021_09_08_tsumura_etal_journalofneuroscience_2021/>[Article] Cross-Hemispheric Complementary Prefrontal Mechanisms during Task Switching under Perceptual Uncertainty.</a></div><div class=mt-2><p>Summary: This paper examines neural mechanisms for task switching in which task-relevant information involved perceptual uncertainty. As a result, the lateral prefrontal cortex (PFC) in the left hemisphere is associated with behavioral flexibility, and the lateral prefrontal cortex (PFC) in the right hemisphere is associated with the perception of ambiguous stimuli.</p><p><a href=https://www.jneurosci.org/content/jneuro/41/10/2197.full.pdf target=_blank rel=noopener>Tsumura, Aoki, et al. (2021). “Cross-Hemispheric Complementary Prefrontal Mechanisms during Task Switching under Perceptual Uncertainty.” The Journal of Neuroscience, 41(10), 2197–2213.</a><br></p></div><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>Sep 8, 2021</span></div></div></div><div class=ml-3></div></div><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/interesting_works/2021_09_01_zhang_etal_journalofbiomedicalandhealthinformatics_2020/>[Article] Canonical Correlation Analysis of Imaging Genetics Data Based on Statistical Independence and Structural Sparsity.</a></div><div class=mt-2><p>Summary: They proposed Independence and Structural sparsity canonical correlation analysis (ISCCA) for imaging genetics study. They combined ICA and CCA to reduce the collinear effects, which also incorporate graph structure of the data into the model to improve the accuracy of feature selection. This method helps identify risk genes and abnormal brain regions in schizophrenia.</p><p><a href=https://ieeexplore.ieee.org/abstract/document/8990142 target=_blank rel=noopener>Zhang, Yipu, et al. &ldquo;Canonical Correlation Analysis of Imaging Genetics Data Based on Statistical Independence and Structural Sparsity.&rdquo; IEEE journal of biomedical and health informatics 24.9 (2020): 2621-2629.</a><br></p></div><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>Sep 1, 2021</span></div></div></div><div class=ml-3></div></div><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/interesting_works/2021_08_25_dwivedi_etal_nature_2020/>[Article] Deriving disease modules from the compressed transcriptional space embedded in a deep autoencoder.</a></div><div class=mt-2><p>Summary: Using deep neural networks(DNN) to classify diseases using genetic data is popular these days. In this article, they hypothesized that disease-relevant modules of genes can be discovered within the autoencoder (AE) representations. They compared shallow and deep AE with various node sizes, and showed deep AE works better. Also, they also showed that each different layer captures gradients of biology. By overlapping top 1000 genes for each disease with GWAS, they found a highly significant association for at least one layer in all tested diseases.</p><p><a href=https://www.nature.com/articles/s41467-020-14666-6 target=_blank rel=noopener>Dwivedi, S. K., Tjärnberg, A., Tegnér, J., & Gustafsson, M. (2020). Deriving disease modules from the compressed transcriptional space embedded in a deep autoencoder. Nature communications, 11(1), 1-10.</a><br></p></div><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>Aug 25, 2021</span></div></div></div><div class=ml-3></div></div><nav class=mt-1><ul class="pagination justify-content-center"><li class=page-item><a class=page-link href=/interesting_works/page/3/>&#171;</a></li><li class=page-item><a class=page-link href=/interesting_works/page/5/>&#187;</a></li></ul></nav></div></div><div class=page-footer><div class=container><footer class=site-footer><p class=powered-by>© 2023 BSPL</p><p class=powered-by>Published with <a href="https://wowchemy.com/?utm_campaign=poweredby" target=_blank rel=noopener>Wowchemy</a> — the free, <a href=https://github.com/wowchemy/wowchemy-hugo-modules target=_blank rel=noopener>open source</a> website builder that empowers creators.</p></footer></div></div><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code class="tex hljs"></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i> Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i> Download</a><div id=modal-error></div></div></div></div></div><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/instant.page/5.1.0/instantpage.min.js integrity="sha512-1+qUtKoh9XZW7j+6LhRMAyOrgSQKenQ4mluTR+cvxXjP1Z54RxZuzstR/H9kgPXQsVB8IW7DMDFUJpzLjvhGSQ==" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/highlight.min.js integrity="sha512-TDKKr+IvoqZnPzc3l35hdjpHD0m+b2EC2SrLEgKDRWpxf2rFCxemkgvJ5kfU48ip+Y+m2XVKyOCD85ybtlZDmw==" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/languages/r.min.js crossorigin=anonymous></script>
<script id=search-hit-fuse-template type=text/x-template>
        <div class="search-hit" id="summary-{{key}}">
          <div class="search-hit-content">
            <div class="search-hit-name">
              <a href="{{relpermalink}}">{{title}}</a>
              <div class="article-metadata search-hit-type">{{type}}</div>
              <p class="search-hit-description">{{snippet}}</p>
            </div>
          </div>
        </div>
      </script><script src=https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin=anonymous></script>
<script src=/js/bootstrap.bundle.min.6aed84840afc03ab4d5750157f69c120.js></script>
<script src=/en/js/wowchemy.min.2cc80485e7b9001edba5cdf5b39a1f97.js></script></body></html>