<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Wowchemy 5.3.0 for Hugo"><meta name=description content><link rel=alternate hreflang=en-us href=https://bspl.korea.ac.kr/interesting_works/><link rel=preconnect href=https://fonts.gstatic.com crossorigin><meta name=theme-color content="#1565c0"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin=anonymous media=print onload='this.media="all"'><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css integrity="sha256-FMvZuGapsJLjouA6k7Eo2lusoAX9i0ShlWFG6qt7SLc=" crossorigin=anonymous media=print onload='this.media="all"'><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/github.min.css crossorigin=anonymous title=hl-light media=print onload='this.media="all"'><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/dracula.min.css crossorigin=anonymous title=hl-dark media=print onload='this.media="all"' disabled><link rel=preload as=style href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap"><link rel=stylesheet href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap" media=print onload='this.media="all"'><link rel=stylesheet href=/css/wowchemy.d4217de0a97b753d87af942d59c81115.css><link rel=alternate href=/interesting_works/index.xml type=application/rss+xml title="Brain Signal Processing Lab"><link rel=manifest href=/manifest.webmanifest><link rel=icon type=image/png href=/media/icon_hu746e86b219933ead05b970d8e60fbf48_161589_32x32_fill_lanczos_center_3.png><link rel=apple-touch-icon type=image/png href=/media/icon_hu746e86b219933ead05b970d8e60fbf48_161589_180x180_fill_lanczos_center_3.png><link rel=canonical href=https://bspl.korea.ac.kr/interesting_works/><meta property="twitter:card" content="summary"><meta property="og:site_name" content="Brain Signal Processing Lab"><meta property="og:url" content="https://bspl.korea.ac.kr/interesting_works/"><meta property="og:title" content="Interesting Works | Brain Signal Processing Lab"><meta property="og:description" content><meta property="og:image" content="https://bspl.korea.ac.kr/media/icon_hu746e86b219933ead05b970d8e60fbf48_161589_512x512_fill_lanczos_center_3.png"><meta property="twitter:image" content="https://bspl.korea.ac.kr/media/icon_hu746e86b219933ead05b970d8e60fbf48_161589_512x512_fill_lanczos_center_3.png"><meta property="og:locale" content="en-us"><meta property="og:updated_time" content="2023-06-14T00:00:00+00:00"><title>Interesting Works | Brain Signal Processing Lab</title></head><body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents class=page-wrapper data-wc-page-id=6968ecbcb5e980dded5666bc95f80948><script src=/js/wowchemy-init.min.1d309c6b3f55725f8869af2651084961.js></script><aside class=search-modal id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=# aria-label=Close><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control aria-label=Search...></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><div class=page-header><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container-xl><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=/>Brain Signal Processing Lab</a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar-content aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=/>Brain Signal Processing Lab</a></div><div class="navbar-collapse main-menu-item collapse justify-content-start" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=/aims><span>BSPL is..</span></a></li><li class=nav-item><a class=nav-link href=/members><span>Members</span></a></li><li class="nav-item dropdown"><a href=# class="nav-link dropdown-toggle" data-toggle=dropdown aria-haspopup=true><span>Publications</span><span class=caret></span></a><div class=dropdown-menu><a class=dropdown-item href=/publications/articles><span>Articles</span></a>
<a class=dropdown-item href=/publications/domestic_conferences><span>Domestic Conferences</span></a>
<a class=dropdown-item href=/publications/international_conferences><span>International Conferences</span></a></div></li><li class="nav-item dropdown"><a href=# class="nav-link dropdown-toggle" data-toggle=dropdown aria-haspopup=true><span>Softwares</span><span class=caret></span></a><div class=dropdown-menu><a class=dropdown-item href=/softwares/dnn><span>DNN</span></a>
<a class=dropdown-item href=/softwares/idr><span>iDR</span></a>
<a class=dropdown-item href=/softwares/rspca><span>rsPCA</span></a>
<a class=dropdown-item href=/softwares/env><span>ENV</span></a></div></li><li class=nav-item><a class="nav-link active" href=/interesting_works><span>Interesting Works</span></a></li><li class=nav-item><a class=nav-link href=/news><span>Lab News</span></a></li><li class=nav-item><a class=nav-link href=/contact><span>Contact Us</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class=nav-item><a class="nav-link js-search" href=# aria-label=Search><i class="fas fa-search" aria-hidden=true></i></a></li><li class="nav-item dropdown theme-dropdown"><a href=# class=nav-link data-toggle=dropdown aria-haspopup=true aria-label="Display preferences"><i class="fas fa-moon" aria-hidden=true></i></a><div class=dropdown-menu><a href=# class="dropdown-item js-set-theme-light"><span>Light</span></a>
<a href=# class="dropdown-item js-set-theme-dark"><span>Dark</span></a>
<a href=# class="dropdown-item js-set-theme-auto"><span>Automatic</span></a></div></li></ul></div></nav></div><div class=page-body><div class="universal-wrapper pt-3"><h1>Interesting Works</h1></div><div class=universal-wrapper><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/interesting_works/2016_12_26_news_a_delicate_balance_between_positive_and_negative_emotion/>[News] A delicate balance between positive and negative emotion</a></div><div class=mt-2><p><figure><div class="d-flex justify-content-center"><div class=w-100><img src="https://news.mit.edu/sites/default/files/styles/news_article__image_gallery/public/images/201610/MIT-emo-balance_0.jpg?itok=BoC1Y85I#50" alt loading=lazy data-zoomable></div></div></figure></p><p>(figure from MITnew)</p><p>Our emotional state is governed partly by a tiny brain structure known as the amygdala, which is responsible for processing positive emotions such as happiness, and negative ones such as fear and anxiety.</p><p>A new study from MIT finds that these emotions are controlled by two populations of neurons that are genetically programmed to encode memories of either fearful or pleasurable events. Furthermore, these sets of cells inhibit each other, suggesting that an imbalance between these populations may be responsible for disorders such as depression and post-traumatic stress disorder.</p><p>If you&rsquo;re interested in, please click here.</p><p><a href=http://news.mit.edu/2016/two-neuron-populations-encode-happy-fearful-memories-1017 target=_blank rel=noopener>http://news.mit.edu/2016/two-neuron-populations-encode-happy-fearful-memories-1017</a></p></div><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>Dec 26, 2016</span></div></div></div><div class=ml-3></div></div><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/interesting_works/2016_12_05_news_a_radiation-free_approach_to_imaging_molecules_in_the_brain/>[News] A radiation-free approach to imaging molecules in the brain</a></div><div class=mt-2><p><figure><div class="d-flex justify-content-center"><div class=w-100><img src="https://news.mit.edu/sites/default/files/styles/news_article__image_gallery/public/images/201612/MIT-Molecular-Imaging_0.jpg?itok=e_ulBB3p#50" alt loading=lazy data-zoomable></div></div></figure></p><p>(figure from MIT news)</p><p>Scientists hoping to get a glimpse of molecules that control brain activity have devised a new probe that allows them to image these molecules without using any chemical or radioactive labels. Currently the gold standard approach to imaging molecules in the brain is to tag them with radioactive probes. However, these probes offer low resolution and they can&rsquo;t easily be used to watch dynamic events, says Alan Jasanoff, an MIT professor of biological engineering and brain and cognitive sciences.</p><p>Jasanoff and his colleagues have developed new sensors consisting of proteins designed to detect a particular target, which causes them to dilate blood vessels in the immediate area. This produces a change in blood flow that can be imaged with magnetic resonance imaging (MRI) or other imaging techniques.</p><p>If you&rsquo;re interested in,
please click the following link.</p><p><a href=http://news.mit.edu/2016/radiation-free-approach-imaging-molecules-brain-mri-1202 target=_blank rel=noopener>http://news.mit.edu/2016/radiation-free-approach-imaging-molecules-brain-mri-1202</a></p></div><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>Dec 5, 2016</span></div></div></div><div class=ml-3></div></div><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/interesting_works/2016_11_21_news_where_music_and_technology_unite/>[News] Where music and technology unite</a></div><div class=mt-2><p><figure><div class="d-flex justify-content-center"><div class=w-100><img src="https://news.mit.edu/sites/default/files/styles/news_article__image_gallery/public/images/201611/founder-journey-zolot-mit.jpg?itok=pm53JcdV#50" alt loading=lazy data-zoomable></div></div></figure></p><p>(figure from MIT news)</p><p>In Developing Technologies for Music and Health, students design and prototype devices that explore music&rsquo;s impact on health and brain functions, such as sleep, anxiety, athletic performance, pain, and even dementia. Music therapy is a well-established field, but the idea of integrating technology and data analysis into music therapy to improve a person&rsquo;s well-being is where the future lies. The therapeutic, clinical, and technical applications of music in health are far-reaching, and our ever-connected world presents an exciting opportunity for the young entrepreneur.</p><p>If you&rsquo;re interested in,
please click the following link.</p><p><a href=http://news.mit.edu/2016/where-music-and-technology-unite-1103 target=_blank rel=noopener>http://news.mit.edu/2016/where-music-and-technology-unite-1103</a></p></div><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>Nov 21, 2016</span></div></div></div><div class=ml-3></div></div><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/interesting_works/2016_11_03_news_mapping_serotonin_dynamics_in_the_living_brain/>[News] Mapping serotonin dynamics in the living brain</a></div><div class=mt-2><p><figure><div class="d-flex justify-content-center"><div class=w-100><img src="https://news.mit.edu/sites/default/files/styles/news_article__image_gallery/public/images/201610/MIT-Mapping-Serotonin_0.jpg?itok=xE6o6oDY#50" alt loading=lazy data-zoomable></div></div></figure></p><p>(Figure from MIT news)</p><p>Serotonin is a neurotransmitter that&rsquo;s partly responsible for feelings of happiness and for mood regulation in humans. This makes it a common target for antidepressants, which block serotonin from being reabsorbed by neurons after it has dispatched its signal, so more of it stays floating around the brain.</p><p>MIT researchers have developed an imaging technique that, for the first time, enables three-dimensional mapping of serotonin as it&rsquo;s reabsorbed into neurons, across multiple regions of the living brain. This technique, the researchers say, gives an unprecedented view of serotonin dynamics, and could be a powerful tool for the research and development of antidepressants.</p><p>If you are interested in, please click the following link.</p><p><a href=http://news.mit.edu/2016/mapping-serotonin-dynamics-living-brain-1020 target=_blank rel=noopener>http://news.mit.edu/2016/mapping-serotonin-dynamics-living-brain-1020</a></p></div><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>Nov 3, 2016</span></div></div></div><div class=ml-3></div></div><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/interesting_works/2016_10_13_news_monitoring_parkinsons_symptoms_at_home/>[News] Monitoring Parkinson's symptoms at home</a></div><div class=mt-2><p><figure><div class="d-flex justify-content-center"><div class=w-100><img src="https://news.mit.edu/sites/default/files/styles/news_article__image_gallery/public/images/201610/MIT-Typing-Parkinsons_0.jpg?itok=gjVblHXH#50" alt loading=lazy data-zoomable></div></div></figure></p><p>(Figure from MIT news)</p><p>Parkinson&rsquo;s disease is the second most common neurodegenerative disorder in the developed world, with around 60,000 people diagnosed in the U.S. each year. Although there is no cure for the disease, there are treatments that can reduce the severity of a patient&rsquo;s symptoms. But for these treatments to be effective, clinicians need a method to regularly monitor the patient&rsquo;s symptoms in the home.</p><p>In a paper published today in the journal <em>Scientific Reports</em>, researchers at MIT and elsewhere describe a technique they have developed to monitor Parkinson&rsquo;s disease progression as patients interact with a computer keyboard.</p><p>If you are interested in, please click the following link</p><p><a href=http://news.mit.edu/2016/keyboard-monitoring-parkinsons-symptoms-1005 target=_blank rel=noopener>http://news.mit.edu/2016/keyboard-monitoring-parkinsons-symptoms</a></p></div><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>Oct 13, 2016</span></div></div></div><div class=ml-3></div></div><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/interesting_works/2016_09_27_news_newly_discovered_neural_connections_may_be_linked_to_emotional_decision-making/>[news] Newly discovered neural connections may be linked to emotional decision-making</a></div><div class=mt-2><p><figure><div class="d-flex justify-content-center"><div class=w-100><img src="https://news.mit.edu/sites/default/files/styles/news_article__image_gallery/public/images/201609/MIT-Dopamine-Systems-1_0.jpg?itok=GifPFDx5#50" alt loading=lazy data-zoomable></div></div></figure></p><p>(Figure from MIT news)</p><p>MIT neuroscientists have discovered connections deep within the brain that appear to form a communication pathway between areas that control emotion, decision-making, and movement. The researchers suspect that these connections, which they call striosome-dendron bouquets, may be involved in controlling how the brain makes decisions that are influenced by emotion or anxiety. This circuit may also be one of the targets of the neural degeneration seen in Parkinson&rsquo;s disease.</p><p>They were able to find these connections using a technique  developed at MIT known as expansion microscopy, which enables scientists to expand brain tissue before imaging it. This produces much higher-resolution images than would otherwise be possible with conventional microscopes.</p><p>If you interested in, please click the following link.</p><p><a href=http://news.mit.edu/2016/neural-connections-linked-emotional-decision-making-0919 target=_blank rel=noopener>http://news.mit.edu/2016/neural-connections-linked-emotional-decision-making-0919</a></p></div><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>Sep 27, 2016</span></div></div></div><div class=ml-3></div></div><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/interesting_works/2016_09_05_news_study_finds_brain_connections_key_to_reading/>[News] Study finds brain connections key to reading</a></div><div class=mt-2><p><figure><div class="d-flex justify-content-center"><div class=w-100><img src="https://news.mit.edu/sites/default/files/styles/news_article__image_gallery/public/images/201608/MIT-Brain-Function_0.jpg?itok=RIyEnX3S#50" alt loading=lazy data-zoomable></div></div></figure></p><p> (Figure from MIT news)</p><p>The researchers from MIT team found that they could predict the child&rsquo;s visual word form area (VMFA), who was in before learning reading. They used functional magnetic resonance imaging (fMRI), and diffusion-weighted imaging in order to trace the connection between the VMFA and other brain area. The researchers recruited some children, and took the scan twice. One is when the children were 5 years old (i.e. before learning reading), and the other one is when they were 8 years old (i.e. after learning reading). Even the child who doesn&rsquo;t know how to read (so there is no VWFA), there are pre-existing connectivity pattern for the VWFA which has different connectivity compare to other brain area. Thus, from this result, the researchers suggest that it could helpful to identify the risk of developing dyslexia or other reading difficulties.</p><p>If you interested in, please click the following link.</p><p><a href=http://news.mit.edu/2016/brain-connections-key-reading-0808 target=_blank rel=noopener>http://news.mit.edu/brain_connections_key_reading</a></p></div><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>Sep 5, 2016</span></div></div></div><div class=ml-3></div></div><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/interesting_works/2016_08_02_news_updated_brain_map_identifies_nearly_100_new_regions/>[News] Updated Brain Map Identifies Nearly 100 New Regions</a></div><div class=mt-2><p><figure id=figure-figure-a-new-map-based-on-brain-scan-data-collected-by-the-human-connectome-project-matthew-f-glasser-david-c-van-essen><div class="d-flex justify-content-center"><div class=w-100><img src=https://static01.nyt.com/images/2016/07/21/science/21ZIMMER1/21ZIMMER1-master768.jpg alt="(Figure. A new map based on brain scan data collected by the Human Connectome Project. Matthew F. Glasser, David C. Van Essen)" loading=lazy data-zoomable></div></div><figcaption>(Figure. A new map based on brain scan data collected by the Human Connectome Project. Matthew F. Glasser, David C. Van Essen)</figcaption></figure></p><p>There were unknown regions. However, by the Human Connectome Project, the researchers could identify around 100 new brain regions. To identify brain regions, the researchers used advanced scanners and artificial intelligence programs. In addition, they recorded high-resolution images, during the tests on memory, language and other tasks relate to thought. The researchers hope that these investigation, the updated brain map, will help their further work, and the further work from these identifications would be finding the connection between the brain regions.</p><p>If you interested in, please click the following link.</p><p><a href="http://www.nytimes.com/2016/07/21/science/human-connectome-brain-map.html?_r=0" target=_blank rel=noopener>http://www.nytimes.com/human_connectome_brain_map</a></p></div><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>Aug 2, 2016</span></div></div></div><div class=ml-3></div></div><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/interesting_works/2016_07_27_news_imaging_the_brain_at_multiple_size_scales/>[News] Imaging the brain at multiple size scales</a></div><div class=mt-2><p><figure><div class="d-flex justify-content-center"><div class=w-100><img src="https://news.mit.edu/sites/default/files/styles/news_article__image_gallery/public/images/201607/MIT-Brain-Map_0.jpg?itok=CA7IgtGE#50" alt loading=lazy data-zoomable></div></div></figure></p><p> (Image: MIT News)</p><p>The researchers in MIT developed a new technique called magnified analysis of proteome (MAP). It could reveal subcellular details and long-range connections. Thus, it might help to investigate the connectivity and functions of neurons in the brain. Also, the MAP imaging technique has high resolution that we could trace the connections in the brain more accurately.</p><p>If you interested in, please click the following link.</p><p><a href=http://news.mit.edu/2016/imaging-brain-multiple-size-scales-0725 target=_blank rel=noopener>http://news.mit.edu/imaging_brain_multiple_size_scales</a></p></div><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>Jul 27, 2016</span></div></div></div><div class=ml-3></div></div><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/interesting_works/2016_07_08_news_unmasking_alzheimers_risk_in_young_adults/>[News] Unmasking Alzheimer's risk in young adults</a></div><div class=mt-2><p><figure><div class="d-flex justify-content-center"><div class=w-100><img src=https://cdn.cnn.com/cnnnext/dam/assets/150325115244-02-alzheimers-woman-super-169.jpg#50 alt=http://edition.cnn.com/2016/07/06/health/alzheimers-risk-young-adults/index.html loading=lazy data-zoomable></div></div></figure></p><p>  (Image from CNN)</p><p>   A study, from Elizabeth C. Mormino and her colleague, suggests that it is possible to detect the risk factors for sporadic Alzheimer&rsquo;s in young adulthood. One way to detect the risk is check images of a patient&rsquo;s hippocampus. If the patient has a small hippocampal volume, it connotes that the patient has various genetic risk for Alzheimer&rsquo;s. The study was performed with 166 people with dementia, and 1,026 people without dementia (mean ages 75 years old), and used MRI images for analyzing. Also researchers checked DNA of the patients, and they calculated risk score about gene variants, which are associated with a high risk of Alzheimer&rsquo;s disease. After that, the researchers checked the risk factors (risk scores and hippocampal volume) with 1,322 healthy adults the ages between 18 and 35. Finally, the researchers found, that there are association between higher risk score and a smaller hippocampal volume.</p><p>If you interested in, please click the following link.</p><p><a href=http://edition.cnn.com/2016/07/06/health/alzheimers-risk-young-adults/index.html target=_blank rel=noopener>http://edition.cnn.com/Alzheimers_risk_young_adults</a></p></div><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>Jul 8, 2016</span></div></div></div><div class=ml-3></div></div><nav class=mt-1><ul class="pagination justify-content-center"><li class=page-item><a class=page-link href=/interesting_works/page/8/>&#171;</a></li><li class=page-item><a class=page-link href=/interesting_works/page/10/>&#187;</a></li></ul></nav></div></div><div class=page-footer><div class=container><footer class=site-footer><p class=powered-by>© 2023 BSPL</p><p class=powered-by>Published with <a href="https://wowchemy.com/?utm_campaign=poweredby" target=_blank rel=noopener>Wowchemy</a> — the free, <a href=https://github.com/wowchemy/wowchemy-hugo-modules target=_blank rel=noopener>open source</a> website builder that empowers creators.</p></footer></div></div><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code class="tex hljs"></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i> Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i> Download</a><div id=modal-error></div></div></div></div></div><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/instant.page/5.1.0/instantpage.min.js integrity="sha512-1+qUtKoh9XZW7j+6LhRMAyOrgSQKenQ4mluTR+cvxXjP1Z54RxZuzstR/H9kgPXQsVB8IW7DMDFUJpzLjvhGSQ==" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/highlight.min.js integrity="sha512-TDKKr+IvoqZnPzc3l35hdjpHD0m+b2EC2SrLEgKDRWpxf2rFCxemkgvJ5kfU48ip+Y+m2XVKyOCD85ybtlZDmw==" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/languages/r.min.js crossorigin=anonymous></script>
<script id=search-hit-fuse-template type=text/x-template>
        <div class="search-hit" id="summary-{{key}}">
          <div class="search-hit-content">
            <div class="search-hit-name">
              <a href="{{relpermalink}}">{{title}}</a>
              <div class="article-metadata search-hit-type">{{type}}</div>
              <p class="search-hit-description">{{snippet}}</p>
            </div>
          </div>
        </div>
      </script><script src=https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin=anonymous></script>
<script src=/js/bootstrap.bundle.min.6aed84840afc03ab4d5750157f69c120.js></script>
<script src=/en/js/wowchemy.min.5bb41a68c539a638856544de9689bd9c.js></script></body></html>